<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>224</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>XB2AssetTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="allTabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>541</width>
      <height>191</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="extractTab">
     <attribute name="title">
      <string>Extract</string>
     </attribute>
     <widget class="QLabel" name="inputLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Input File(s):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputFiles">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>451</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Input files...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="inputBrowse">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
      <property name="icon">
       <iconset theme="DirIcon">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="outputBrowse">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
      <property name="icon">
       <iconset theme="DirIcon">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="outputLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Output Directory:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="outputDir">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>451</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Output directory...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="extractButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>100</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Extract</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="saveXbc1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Save raw XBC1 files</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="logTab">
     <attribute name="title">
      <string>Log</string>
     </attribute>
     <widget class="QTextEdit" name="debugConsole">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>531</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="enableVerbose">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable verbose messages (For debugging purposes)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveLog">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>140</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Log...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="aboutTab">
     <attribute name="title">
      <string>About</string>
     </attribute>
     <widget class="QLabel" name="aboutxb2at">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Todo add a semi-meaningful about section
(also mention rewrite)</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="aboutQtButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>About Qt...</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="appName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>XB2AssetTool</string>
    </property>
   </widget>
   <widget class="QLabel" name="appDesc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TODO: Add a description here</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="xb2at.qrc"/>
 </resources>
 <connections/>
</ui>
